app/build.gradle

```diff
@@ -104,6 +106,8 @@ android {
             resValue "string", "dropbox_app_key", "dhjh8ke9wf05948"
             resValue "string", "dropbox_app_secret", "h2t9fphj3nr4wkw"
             resValue "string", "manifest_dropbox_app_key", "db-dhjh8ke9wf05948"
+
+            dimension "stability"
```

Comment:
When there's just one dimension, it's assigned by default to all flavors, you don't need to specify it. See [here](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.AppExtension.html#com.android.build.gradle.AppExtension:flavorDimensions(java.lang.String[])).